tenseal/binding.cpp

```diff
@@ -87,6 +87,12 @@ PYBIND11_MODULE(_tenseal_cpp, m) {
         .def("neg", &CKKSVector::negate)
         .def("neg_", &CKKSVector::negate_inplace)
         .def("neg_inplace", &CKKSVector::negate_inplace)
+        .def("square", &CKKSVector::square)
+        .def("square_", &CKKSVector::square_inplace)
+        .def("square_inplace", &CKKSVector::square_inplace)
+        .def("pow", &CKKSVector::power)
+        .def("pow_", &CKKSVector::power_inplace)
+        .def("pow_inplace", &CKKSVector::power_inplace)
```

Comment:
The python API have have this inplace operations terminated by a `_` (e.g. `add_`) only and we aren't providing `function_name_inplace` kind of methods.